test/mailers/previews/message_preview.rb

```diff
@@ -0,0 +1,9 @@
+# Preview all emails at http://localhost:3000/rails/mailers/message
+class MessagePreview < ActionMailer::Preview
+
+  # Preview this email at http://localhost:3000/rails/mailers/message/contact_us
+  def contact_us
+    Message.contact_us
```

Comment:
From testing this out (using the [mentioned URL](http://localhost:3000/rails/mailers/message/contact_us)) this preview doesn't seem to work. Do you have any idea why that might be?

Reply:
I'll delete this file because it is misleading.  The preview at http://localhost:3000/rails/mailers/contact_mailer works.